vivisect/parsers/pe.py

```diff
@@ -286,7 +287,7 @@ def loadPeIntoWorkspace(vw, pe, filename=None):
                 vw.markDeadData(secbase, secbase+len(secbytes))
 
         except Exception, e:
-            print "Error Loading Section (%s size:%d rva:%.8x offset: %d): %s" % (secname,secfsize,secrva,secoff,e)
+            logging.getLogger("parsers.pe.loadPE").warning("Error Loading Section (%s size:%d rva:%.8x offset: %d): %s", secname, secfsize, secrva, secoff, e)
```

Comment:
i dont think the calls to `logging.getLogger(some arbitrary string)` are very idiomatic, but this PR can serve as a place to discuss the idea of using the `logging` module. probably makes sense to attached a logger instance to each vw, emu, etc. and use those instances directly for logging.